packages/react/src/components/RadioButton/RadioButton.stories.module.css

```diff
@@ -0,0 +1,31 @@
+.parentDiv {
+  display: flex;
+  background: var(--fds-semantic-background-subtle);
+  padding: 100px;
+  border-radius: var(--fds-spacing-2);
+  cursor: pointer;
+  position: relative;
+}
+
+.parentDiv:focus-within {
+  --fds-inner-focus-border-color: #1e2b3c;
+  --fds-outer-focus-border-color: #fadf4b;
+  --fds-focus-border-width: 3px;
+
+  box-shadow: 0 0 0 var(--fds-focus-border-width) var(--fds-inner-focus-border-color);
+  outline: var(--fds-focus-border-width) solid var(--fds-outer-focus-border-color);
+  outline-offset: var(--fds-focus-border-width);
+}
+
+.parentDiv span:focus-within {
+  box-shadow: none;
+  outline: none;
+}
```

Comment:
The following CSS should be removed because it does not belong to the implementation of renderLabelAsSpan. This is just CSS to display an example within Storybook. Suggest simplifying the example within the storybook to avoid all this extra CSS.

I think we can delete this CSS file. :)

```suggestion

```

Reply:
ah, I did not catch that this was a .stories.css. 😅 Anyway by implementing ref instead of renderLabelAsSpan then we do not need this example. :)